### PR TITLE
Add uwp-setup to mirrored repos.

### PR DESCRIPTION
**Pull request by crummel**

### CODE REVIEW

**Comment by crummel** [comment]
@dagood doesn't look like I can ping reviewers on this, would you be the right person to take a look?

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -509,7 +509,10 @@
     {
       "triggerPaths": [
         "https://github.com/dotnet/arcade/blob/master/**/*",
-        "https://github.com/dotnet/machinelearning/blob/master/**/*"
+        "https://github.com/dotnet/machinelearning/blob/master/**/*",
+        "https://github.com/dotnet/uwp-setup/blob/master/**/*",
+        "https://github.com/dotnet/uwp-setup/blob/rel/6.0/**/*",
+        "https://github.com/dotnet/uwp-setup/blob/rel/6.1/**/*",
```
Comment (dagood) [inline Maestro/subscriptions.json]: Why not `https://github.com/dotnet/uwp-setup/blob/rel/**/*`?

Comment (dagood) [inline Maestro/subscriptions.json]: They also need the `arm64` branch mirrored. I'll add more context to the core-eng issue.

Comment (crummel) [reply Maestro/subscriptions.json]: Good point, changed.

Comment (crummel) [reply Maestro/subscriptions.json]: Added arm64.

**Pull request merged**